openmc/settings.py

```diff
@@ -69,11 +69,17 @@ class SettingsFile(object):
     cross_sections : str
         Indicates the path to an XML cross section listing file (usually named
         cross_sections.xml). If it is not set, the :envvar:`CROSS_SECTIONS`
-        environment variable will be used to find the path to the XML cross
-        section listing.
+        environment variable will be used for continuous-energy calculations
+        and :envvar:`MG_CROSS_SECTIONS` will be used for multi-group
+        calculations to find the path to the XML cross section file.
     energy_grid : str
         Set the method used to search energy grids. Acceptable values are
         'nuclide', 'logarithm', and 'material-union'.
+    energy_mode : str
```

Comment:
`energy_mode : {'continuous-energy', 'multi-group'}`

then you can remove statement about acceptable values